src/api_server/src/crypto.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2022 by Alibaba.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Could we call that file `resource.rs`? And then make it focus on generating resources:

- `fn secret_resource(tee_pub_key: TeePubKey, resource_byte: Vec<u8>) -> Result<Response>`
- `fn token_resource(tee_pub_key: TeePubKey) -> Result<Token>`

The fact that we're using crypto to generate those resources is an implementation detail imho.